oauth2_provider/models.py

```diff
@@ -629,6 +636,53 @@ def revoke(self):
         """
         self.delete()
 
+    def send_backchannel_logout_request(self, ttl=timedelta(minutes=10)):
```

Comment:
Why put the `send_backchannel_logout_request`  method on AbstractIdToken? It doesn't operate on the token and the token is not an entity I think of as being responsible for sending requests. I feel like it's responsibility it carrying the identity data. A standalone function in handlers.py that receives an id_token as an argument would be more straight forward approach and would keep the handler bundled with where it is used for easier discoverability if someone were to want to extend it's scope by wrapping it in a proxy function and overriding it.